instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/NetServerAttributesGetter.java

```diff
@@ -20,9 +20,21 @@
   @Nullable
   String transport(REQUEST request);
 
+  /**
+   * @deprecated implement {@link #sockPeerAddr(Object)} instead.
+   */
+  @Deprecated
   @Nullable
-  Integer peerPort(REQUEST request);
+  default String peerIp(REQUEST request) {
+    return null;
+  }
 
   @Nullable
-  String peerIp(REQUEST request);
+  Integer sockPeerPort(REQUEST request);
```

Comment:
Could we have a deprecation cycle for that one too?

Reply:
see above(?)

Reply:
`peerPort()` was removed, right? (Or am I missing something because it's morning and I haven't had enough coffee yet 😂 )

Reply:
🤦‍♂️

Reply:
added